### PR TITLE
isNumeric: Updated description based on new behavior

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fixes gh-817

### CODE REVIEW

**Comment by AurelioDeRosa** [comment]
@timmywil Does it match what you had in mind?

**Comment by AurelioDeRosa** [comment]
@timmywil friendly ping.

File: entries/jQuery.isNumeric.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
   </signature>
   <longdesc>
     <p>The <code>$.isNumeric()</code> method checks whether its argument represents a numeric value. If so, it returns <code>true</code>. Otherwise it returns <code>false</code>. The argument can be of any type.</p>
+    <p>As of jQuery 3.0 <code>$.isNumeric()</code> returns <code>true</code> only if the argument is of type <code>number</code>, or if it's of type <code>string</code> and it can be coerced into finite numbers. In all other cases, it returns <code>false</code>.</p>
```
Comment (arthurvr) [inline entries/jQuery.isNumeric.xml]: Maybe link to the `types` page to avoid confusion (http://api.jquery.com/Types/#String)

**Comment by timmywil** [comment]
Looks good.

**Pull request closed without merging**